website/layouts/_shortcodes/tabs.html

```
@@ -0,0 +1,66 @@
+    -> #prerequisites-sigstore). The first occurrence keeps the plain ID so
+    existing published links stay valid.
+-->
+
+{{ if .Inner }}{{ end }}
+{{ $id := .Get 0 }}
+{{ $group := printf "tabs-%s" $id }}
+
+<nav>
+  <div class="nav nav-tabs" id="nav-tab" role="tablist">
+  {{ range $index, $tab := .Scratch.Get $group -}}
+    <button data-toggle-tab="{{ $tab.Name | lower }}" class="nav-link{{ if not $index }} active{{ end }}" id="{{ printf "%s-%d" $group $index }}-tab" data-bs-toggle="tab" data-bs-target="#{{ printf "%s-%d" $group $index }}" type="button" role="tab" aria-controls="{{ printf "%s-%d" $group $index }}" aria-selected="true">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)